### PR TITLE
Fix to #1528 - Exception thrown for multi-level include dotting over one-one navigation into one to many deeper inside the include chain

### PR DESCRIPTION
**Pull request by maumar**
Problem was that when we tried to extract list of PropertyInfos that include chain consists of, we were doing it incorrectly - always assuming it would be a simple property (or list of those inside anonymous type).
Fix is to use a proper method that expects complex properties instead.

### CODE REVIEW

**Comment by maumar** [comment]
@anpete

File: src/EntityFramework.Core/Utilities/ExpressionExtensions.cs [hunk 1]
```diff
@@ -89,6 +89,34 @@ var propertyPath
             return propertyInfos != null && propertyInfos.Length == 1 ? propertyInfos[0] : null;
         }
 
+        public static PropertyInfo[] GetComplexPropertyAccess(this LambdaExpression propertyAccessExpression)
```
Comment (maumar) [inline src/EntityFramework.Core/Utilities/ExpressionExtensions.cs]: will add [NotNull]

**Comment by anpete** [comment]
:shipit:

**Comment by maumar** [comment]
merged c12501a2d884d240f83de7a6b236c9a37af28db2

**Pull request closed without merging**